android/src/main/java/com/revenuecat/purchases/common/subscriberAttributes.kt

```diff
@@ -23,6 +24,7 @@ fun addAttributionData(
 ) {
     for (attributionNetwork in Purchases.AttributionNetwork.values()) {
         if (attributionNetwork.serverValue == network) {
+            @Suppress("DEPRECATION")
```

Comment:
suppressed since we're already declaring this as deprecated, but we have no choice but to internally call the deprecated method